core/src/main/resources/error/error-classes.json

```diff
@@ -1328,6 +1328,12 @@
       "Unable to convert SQL type <toType> to Protobuf type <protobufType>."
     ]
   },
+  "UNABLE_TO_INFER_SCHEMA" : {
+    "message" : [
+      "Unable to infer schema for <format>. It must be specified manually."
+    ],
+    "sqlState" : "46110"
```

Comment:
```suggestion
    "sqlState" : "42KD9"
```

Reply:
Updated, thanks!

Btw, seems like we don't have "42KD9" in https://github.com/apache/spark/tree/master/core/src/main/resources/error.

Do we want to update the list ?

Reply:
Yes
https://docs.google.com/spreadsheets/d/1hrQBSuHooiozUNAQTHiYq3WidS1uliHpl9cYfWpig1c/edit?usp=sharing

We should sync once in a while

Reply:
Got it. Let me followup the list periodically.
Thanks! :-)